### PR TITLE
rules.json: rule to trigger the buzzer from the button

### PR DESCRIPTION
**Pull request by gvancuts**
Add a rule to the Home GW rule engine (gateway/rules-engine/rules.json)
to trigger the buzzer when the button (if present) is pressed. This is,
in other words, a basic doorbell :)

Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

### CODE REVIEW

**Comment by gvancuts** [comment]
I haven't specifically tested this but @nagineni 's feedback (in #108) indicated that this works fine.

File: gateway/rules-engine/rules.json [hunk 1]
```diff
@@ -43,4 +43,26 @@
         "remove":{
         }
     }
+    "button":{
```
Comment (nagineni) [inline gateway/rules-engine/rules.json]: I'm sure this will through a parse error due to the missing ','.

```
},
"button":{
```

Comment (gvancuts) [inline gateway/rules-engine/rules.json]: Good catch @nagineni , I have updated the patch but let me test it a little more then to make sure there is no other silly mistake in there ;)

**Comment by gvancuts** [comment]
I've actually tested this now (using a simulated buzzer and button) and the output (using NODE_DEBUG=buzzer and NODE_DEBUG=button) shows that the rule is in effect. This should be good to go in.

**Comment by nagineni** [comment]
LGTM. @gvancuts thanks for testing. :smile:

**Pull request merged**